swift-sdk/Internal/InAppPresenter.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  Created by Jay Kim on 4/21/20.
+//  Copyright © 2020 Iterable. All rights reserved.
+//
+
+import UIKit
+
+class InAppPresenter {
+    static var isPresenting = false
+    
+    private let delayInterval: TimeInterval = 1.0
+    
+    private var topVC: UIViewController
+    private var htmlMessageVC: IterableHtmlMessageViewController
+    private var delayTimer: Timer?
+    
+    init(topViewController: UIViewController, htmlMessageViewController: IterableHtmlMessageViewController) {
+        ITBInfo()
+        topVC = topViewController
```

Comment:
I don't think we need to abbreviate class members. We should use `self.topViewController = topViewController`